### DISCUSSION TITLE
disable online backup or use only local disk

### DISCUSSION THREAD

**Discussion by IlTasso** [anyproto/contributors#135] [Q&A] [answered]
Hi everyone, is it possible to disable online backup? Let me explain, I would like to be able to work completely offline so that the data, even if encrypted, does not leave the HD of my PC.
Thank you

**Comment by Seanytype** [comment, marked as answer]
Yes there is local-only mode. 

Check out this [community made tutorial](https://community.anytype.io/t/video-anytype-inital-setup-for-local-only-offline/13123) which covers the initial set up of Anytype.
